sentences/de/_common.yaml

```diff
@@ -162,7 +162,7 @@ expansion_rules:
   artikel_unbestimmt: "[(ein|eine|eines|einer|einem|einen)]"
   artikel: "[<artikel_bestimmt>|<artikel_unbestimmt>]"
   name: "<artikel> {name}"
-  area: "([(in|an|auf)] <artikel>|im|am|<von_dem>) {area}[s]"
+  area: "[([(in|an|auf)] <artikel>|im|am|<von_dem>)] {area}[s]"
```

Comment:
please move this change to a separate PR and explain the intention or goal of this change. further we also need test sentences for this change

Reply:
That should not be mandatory right now. There are many brackets which might be confusing.
I have also changed something here in #1657